curriculum/challenges/english/25-front-end-development/quiz-html-tables-and-forms/66ed902df45ce3ece4053ebc.md

```diff
@@ -83,7 +83,7 @@ It is used to define a table cell.
 
 #### --text--
 
-What's the function of the `colspan` attribute?
+What is the function of the `colspan` attribute?
```

Comment:
The question is clear, but we can make it more natural by rewording it. Instead of `What is the function of the `colspan` attribute?`, we could say:
`What does the colspan attribute do in HTML tables?`

Reply:
I think we're probably fine to go with it like it is.